src/main/java/org/isf/examination/gui/PatientExaminationEdit.java

```diff
@@ -1330,11 +1336,24 @@ private JButton getJButtonDelete() {
 		return jButtonDelete;
 	}
 	
-	private JButton getJButtonCancel() {
+	private JButton getJButtonClose() {
 		if (jButtonCancel == null) {
```

Comment:
shouldn't the button be renamed to `jButtonClose`?

Reply:
Yep, thank you!